src/app/authentication/router/router.go

```diff
@@ -1 +1,33 @@
 package router
+
+import (
+	"authentication/constants"
+	"authentication/handlers"
+	"authentication/service"
+
+	"github.com/gin-gonic/gin"
+)
+
+// router is created with gin
+func InitializeRouter() *gin.Engine {
+	userRouter := gin.New()
+	userRouter.Use(gin.Logger()) // LOG DETAILS ON CONSOLE
+	userRouter.POST(constants.CustomerSignupEndpoint, handlers.RegisterCustomer)
+
+
+
+	// SignIn 
+	isAuthorizedEmail := service.IsAuthorizedEmail
```

Comment:
```suggestion
	userRouter.POST(constants.CustomerSignupEndpoint, handlers.RegisterCustomer)

	// SignIn 
	isAuthorizedEmail := service.IsAuthorizedEmail
```

Single space is more than enough.